gc/base/MemorySubSpaceGeneric.cpp

```diff
@@ -791,6 +791,12 @@ MM_MemorySubSpaceGeneric::removeTenureRange(MM_EnvironmentBase* env, uintptr_t s
 
 uintptr_t
 MM_MemorySubSpaceGeneric::releaseFreeMemoryPages(MM_EnvironmentBase* env)
+{
+	return releaseFreeMemoryPages(env, 0);
+}
+
+uintptr_t
+MM_MemorySubSpaceGeneric::releaseFreeMemoryPages(MM_EnvironmentBase* env, uintptr_t includeMemoryType)
```

Comment:
Again, this method is not needed. The default one (without parameters) should directly call memory pool